modules/microshift-low-latency-config-yaml.adoc

```diff
@@ -0,0 +1,67 @@
+
+The first step in enabling low latency to a {microshift-short} cluster is to add configurations to the {microshift-short} `config.yaml` file.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* You made a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`.
+//anything else? do we need to create the KubeletConfiguration first? or does this config populate that for us?
+
+.Procedure
+
+. Add the kubelet configuration to the {microshift-short} `config.yaml` file
++
+.Example passthrough `kubelet` configuration
+[source,yaml]
+----
+# ...
+kubelet: <1>
+  cpuManagerPolicy: static
+  cpuManagerPolicyOptions:
+    full-pcpus-only: "true"
+  cpuManagerReconcilePeriod: 5s
+  memoryManagerPolicy: Static
+  topologyManagerPolicy: single-numa-node
+  reservedSystemCPUs: 0-1
+  reservedMemory:
+  - limits:
+      memory: 1100Mi <2>
+    numaNode: 0
+  kubeReserved:
+    memory: 500Mi
+  systemReserved:
+    memory: 500Mi
+  evictionHard: <3>
+    imagefs.available: "15%" <4>
+    memory.available: "100Mi" <5>
+    nodefs.available: "10%" <6>
+    nodefs.inodesFree: "5%" <7>
+  evictionPressureTransitionPeriod: 0s
+# ...
+----
+//How many footnotes do we want to include? Any others?
+<1> If you change the CPU or memory managers in the kubelet configuration, you must remove files that cache the previous configuration. Restart to remove them automatically, or manually remove the `- /var/lib/kubelet/cpu_manager_state` and `- /var/lib/kubelet/memory_manager_state` files.
+<2> The value in `reservedMemory[0].limits.memory`, 1100 Mi in this example, is equal to `kubeReserved.memory` + `systemReserved.memory` + `evictionHard.memory.available`.
+<3> The `evictionHard` parameters define under which conditions the kubelet evicts pods. When you change the
+default value of only one parameter for the `evictionHard` stanza, the default values of other parameters are not inherited and are set to zero. Provide all the threshold values even when you want to change just one.
+<4> The `imagefs` is a filesystem that container runtimes use to store container images and container writable layers. In this example, the `evictionHard.imagefs.available` parameter means that the pod is evicted when the available space of the image filesystem is less than 15%.
+<5> In this example, the `evictionHard.memory.available` parameter means that the pod is evicted when the available memory of the node drops below 100MiB.
+<6> In this example, the `evictionHard.nodefs.available` parameter means that the pod is evicted when the main filesystem of the node has less than 10% available space.
+<7> In this example, the `evictionHard.nodefs.inodesFree` parameter means that the pod is evicted when more than 15% of the node's main filesystem's inodes are in use.
+
+//How do we verify that this passed to the kubelet correctly?
```

Comment:
Provided settings will appear in /var/lib/microshift/resources/kubelet/config/config.yaml